src/api.v2/model/Experiment.js

```diff
@@ -143,6 +143,25 @@ class Experiment extends BasicModel {
     }
   }
 
+  async getProcessingConfig(experimentId) {
+    const result = await this.find({ id: experimentId });
+    if (_.isEmpty(result)) {
+      throw new NotFoundError('Experiment not found');
+    }
+
+    return result[0].processingConfig;
```

Comment:
if the query is expected to return a single value use `first()` instead of. `[0]`

Reply:
actually just saw that there's a `findOne` method in the `BasicModel` use that